### DISCUSSION TITLE
Visual Studio for Mac support

### DISCUSSION THREAD

**Discussion by debashishkoirala** [community/community#36470] [Copilot Conversations]
I have loved using Copilot for Visual Studio on my windows machine as well as on VS Code for Mac, but I see that Copilot does not currently have support for Visual Studio for Mac. Is this something in the works? When can we expect full Copilot Mac support including for Visual Studio.

Thanks!

**Comment by iamcjwilliams** [comment]
@debashishkoirala Thank you for your feedback, have you updated to the latest version 1.52.0.0 of the Visual Studio extension?

**Comment by ossanna16** [comment]
Hey there @debashishkoirala and thank you for your feedback! I'm glad you found our community 🙂
